app/controllers/posts_controller.rb

```diff
@@ -9,5 +9,26 @@ def show
     @user_id = params[:user_id]
     @post_id = params[:id]
     @post = Post.find(params[:id])
+    @comment = Comment.new
+    @like = Like.new
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = current_user.posts.new(post_parameters)
+    if @post.save
+      redirect_to user_post_path(current_user, @post)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
```

Comment:
- [ ] I could not create a post, look at this short [video](https://www.loom.com/share/b92018d3ade840f0af3b24035d5e4d89?sid=b780c367-def4-4cfc-9fc6-6ef97e014580) to understand what I mean